lib/rules/no-constant-condition.js

```diff
@@ -144,10 +206,18 @@ module.exports = {
 
         return {
             ConditionalExpression: checkConstantCondition,
+            "ConditionalExpression:exit": checkConstantConditionLoopInSet,
```

Comment:
Is this line necessary? It seems like `ConditionalExpression` nodes would never be in `setOfConstantLoops` anyway.